app/keepers/modules.go

```diff
@@ -28,6 +28,7 @@ import (
 	ica "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts"
 
 	_ "github.com/osmosis-labs/osmosis/v12/client/docs/statik"
+	concentrated_liquidity "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity"
```

Comment:
can we use `concentratedliquidity` here? or change it to use the underscore in app/modules.go. Just unifying it